lib/adapters/QUnitAdapter.js

```diff
@@ -6,67 +6,37 @@ export default class QUnitAdapter extends EventEmitter {
     super()
     this.QUnit = QUnit
     QUnit.begin(this.onBegin.bind(this))
-    QUnit.moduleStart(this.onModuleStart.bind(this))
     QUnit.testStart(this.onTestStart.bind(this))
     QUnit.log(this.onLog.bind(this))
     QUnit.testDone(this.onTestDone.bind(this))
-    QUnit.moduleDone(this.onModuleDone.bind(this))
     QUnit.done(this.onDone.bind(this))
 
     this.testsDetails = {}
-    this.emitGlobal = false
+    this.type = {
```

Comment:
I don't like this object, same issue as the event name abstraction we discussed a while ago. Passing `'start'` and `'end'`as strings will do the trick.